lib/rules/lint-no-obsolete-elements.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const Rule = require('./base');
+
+const errorMessage = element => `Use of <${element}> detected. Do not use deprecated elements.`;
+
+const OBSOLETE_ELEMENTS = [
```

Comment:
Can you put an inline comment just above here with the link to the whatwg listing?